api/managementpb/jobs.proto

```diff
@@ -0,0 +1,98 @@
+syntax = "proto3";
+
+package management;
+
+option go_package = "api/managementpb;managementpb";
+
+import "github.com/mwitkow/go-proto-validators/validator.proto";
+import "google/api/annotations.proto";
+import "google/protobuf/duration.proto";
+
+message GetJobRequest {
+  // Unique Job ID.
+  string job_id = 1 [
+    (validator.field) = {
+      string_not_empty: true
+    }
+  ];
+}
+
+message GetJobResponse {
+  // Error contains job error message.
+  message Error {
+    string message = 1;
+  }
+  // Echo contains result for echo job.
+  message Echo {
+    string message = 1;
+  }
+  // Unique Action ID.
+  string job_id = 1;
+  // pmm-agent ID where this Action is running / was run.
+  string pmm_agent_id = 2;
+  // True if Action is finished.
+  bool done = 4;
+  oneof result {
+    // Error result.
+    Error error = 10;
+    // Echo job result.
+    Echo echo = 11;
+  }
+}
+
+message StartEchoJobRequest {
+  // pmm-agent ID where to run this Action.
+  string pmm_agent_id = 1;
```

Comment:
should it be required?

Reply:
Hm, yes, for echo job it's required. But in general - no.

Fixed.